variable/emissions/emissions.yaml

```diff
@@ -233,7 +261,7 @@ Emissions|BC|Energy|Supply:
                 (IPCC category 1C)
    unit: Mt BC/yr
 
-Emissions|BC|Other:
+Emissions|Black Carbon|Other:
```

Comment:
If there is "other" for me it means that the variable is additive, ie Sum of Emissions|Black carbon|X (X being everything except other) + Emissions|Black Carbon|Other = Emissions|Black Carbon
Then I am not sure of that as there may be some overlaps, eg a part of the energy used for heatingis used by the residential and commercial sector....